docs/articles/workflow/creating-release-process.rst

```diff
@@ -0,0 +1,56 @@
+:orphan:
+
+Creating a release process
+==========================
+
+In order to make sure the software that one releases is of high quality,
+traceable, reproducible and so on, it is important to have a well-defined
+release process. We describe a simple process for how to create a release
+process.
+
+It is important to document the process and the various steps in it. In order to
+be able to improve the actual process, and to keep it transparent we suggest
+that the release process is kept together with the rest of the documentation for
+the project.
+
+Create a list of artifacts
+--------------------------
+To be able to maintain a release, a list of artifacts is required to have. The
+artifacts constitute the product of the release, i.e. the matter that is
+released. This means that the recipient of the release should need nothing more
+than the artifacts (except readily available tooling etc) in order to be able to
+use the release.
+
+We recommend having a file with release notes as one of the artifacts, so a user
+only needs one file to read up on the release contents.
+
+Create a list of component attributes for the release
+-----------------------------------------------------
+These attributes are mostly useful internally for being able to tie the various
+components of the release to the actual release. If components are stored in
+source code repositories, it makes sense to point out a specific revision of the
+source code, for example. Other types of attributes could be naming schemes or
+file name patterns, for example.
```

Comment:
"If components are stored in source code repositories, it makes sense to point out a specific revision of the source code, for example." This sentence is well formed in speech, but not in text. I would rephrase it. Perhaps: "E.g. if components are stored in source code repositories, it makes sense to point out a specific revision of the source code."
Also the same for the last sentence. My suggestion is just to remove "for example".

Reply:
Will do!